app/models/season.rb

```diff
@@ -13,6 +13,10 @@ def succ
       find_or_create_by(name: (Date.today.year+1).to_s)
     end
 
+    def prev
+      find_or_create_by(name: (Date.today.year-1).to_s)
+    end
```

Comment:
I thought we already had a `Season.prev`. Since it's only used in a factory, I suggest to remove the class method again and create the season for the previous year directly in the factory.